### PR TITLE
add timezone, codes and properties in stop_area

### PR DESCRIPTION
**Pull request by xlqian**
https://jira.canaltp.fr/browse/NAVP-726

### CODE REVIEW

File: libs/mimir/src/objects.rs [hunk 1]
```diff
@@ -236,6 +242,12 @@ pub struct Stop {
     pub physical_modes: Vec<PhysicalMode>,
     #[serde(default)]
     pub coverages: Vec<String>,
+    #[serde(default)]
```
Comment (xlqian) [inline libs/mimir/src/objects.rs]: @TeXitoi  is this necessaryN

File: libs/mimir/src/objects.rs [hunk 2]
```diff
@@ -236,6 +242,12 @@ pub struct Stop {
...
+    pub timezone: String,
+    #[serde(default)]
```
Comment (xlqian) [inline libs/mimir/src/objects.rs]: @TeXitoi is this necessary?

File: libs/mimir/src/objects.rs [hunk 3]
```diff
@@ -236,6 +242,12 @@ pub struct Stop {
...
+    pub codes: Vec<Code>,
+    #[serde(default)]
```
Comment (xlqian) [inline libs/mimir/src/objects.rs]: @TeXitoi is this necessary?

File: libs/mimir/src/objects.rs (hunk 1 above)
Comment (TeXitoi) [reply libs/mimir/src/objects.rs]: If new bragi real old database, without that it will crash.

File: libs/mimir/src/objects.rs (hunk 2 above)
Comment (TeXitoi) [reply libs/mimir/src/objects.rs]: same

File: libs/mimir/src/objects.rs (hunk 3 above)
Comment (TeXitoi) [reply libs/mimir/src/objects.rs]: same

**Pull request merged**